package-lock.json

```diff
@@ -9017,9 +9017,9 @@
             }
         },
         "vscode-azureextensionui": {
-            "version": "0.29.6",
-            "resolved": "https://registry.npmjs.org/vscode-azureextensionui/-/vscode-azureextensionui-0.29.6.tgz",
-            "integrity": "sha512-GKRNqdjHwu5qfxGD16Y64Z8t+nOQk4j9FKkipqgZGrnGfqO0v42PinbLWFT7WdD85ebPmmlpyNa4F/nDLV7oMQ==",
+            "version": "0.29.7",
```

Comment:
Odd that the vscode-azureappservice package didn't update.

Reply:
I can't update it because I'm still waiting for https://github.com/microsoft/vscode-azuretools/pull/652 to be approved 😛